packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -216,6 +237,13 @@ export class InsuredBridgeL1Client {
     this.logger.debug({ at: "InsuredBridgeL1Client", message: "Insured bridge l1 client updated" });
   }
 
+  private _getInstantRelayHash(depositHash: string, realizedLpFeePct: string): string | null {
+    const instantRelayDataHash = this.web3.utils.soliditySha3(
```

Comment:
do you want to add `soliditySha3` to the top set of `const { toBN } = Web3.utils;` to clean this up a little bit?

same with `eth.abi.encodeParameters`

Reply:
I can't figure out how to get `encodeParameters`. [docs](https://web3js.readthedocs.io/en/v1.2.6/web3.html#web3-modules) do you see a way? Do we need to import a new `web3-eth` package?